Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -231,6 +258,9 @@ gdjs.LightRuntimeObjectPixiRenderer.prototype.raycastTest = function () {
   var result = [];
   this._manager.getAllObstaclesAround(this._object, this._radius, result);
 
+  // Bail out early if there are no obstales.
+  if(result.length === 0) return result;
```

Comment:
Question:
Should we maybe return null at this point? Does returning an empty array might make it live longer than it needs to be, and would it pollute the scope of another function (`updateBuffers()`) as well?

Reply:
I think it's unlikely to make a huge difference, and returning an empty array is not polluting anything. I think it's fine to keep it! It's less surprising for the caller (you know that you'll get an array in all cases, even if empty in case of no obstacles at all :))